internal/eval/eval.go

```diff
@@ -2570,7 +2570,10 @@ func evalPFADD(args []string, store *dstore.Store) []byte {
 		return clientio.Encode(1, false)
 	}
 
-	existingHll := obj.Value.(*hyperloglog.Sketch)
+	existingHll, ok := obj.Value.(*hyperloglog.Sketch)
+	if !ok {
+		return diceerrors.NewErrWithMessage("*WRONGTYPE*")
```

Comment:
We shall change the error as below to maintain parity with Redis.
`WrongTypeHllErr    = "-WRONGTYPE Key is not a valid HyperLogLog string value."`

```
if !ok {
  return diceerrors.NewErrWithMessage(diceerrors.WrongTypeErr)
}
```

Output from Redis:
```
127.0.0.1:6379> pfadd hll a b c
(integer) 1
127.0.0.1:6379> setrange hll 0 "0123"
(integer) 27
127.0.0.1:6379> pfcount hll
(error) WRONGTYPE Key is not a valid HyperLogLog string value.
```

Reply:
Done. thanks